IPython/html/static/notebook/js/kernelselector.js

```diff
@@ -169,7 +169,8 @@ define([
         var that = this;
         if (typeof selected === 'string') {
             selected = {
-                name: selected
+                name: selected,
+                language: selected
```

Comment:
Well, what would you do ?  There are assumption that selected has 'name' and 'language' a few places further in the code, and would require lots of refactoring.  I assume that if you say `set_kenel('julia')` you can mean both language and name ?